tests/integration-tests/tests/custom_resource/test_cluster_custom_resource.py

```diff
@@ -103,17 +103,14 @@ def test_cluster_update(region, cluster_custom_resource_factory, external_update
     # Update the stack
     update_params = parameters | update_parameters
     stack_params = [{"ParameterKey": k, "ParameterValue": v} for k, v in update_params.items()]
-    stack.factory.update_stack(stack.name, stack.region, stack_params)
+    stack.factory.update_stack(stack.name, stack.region, stack_params, True)
```

Comment:
nitpick, non-blocking: Here and below, I'd use named parameters here for better clarity i.e. `stack_under_test=True`

Reply:
Ok will do on next PR